api/controllers/auth.ts

```diff
@@ -29,10 +28,10 @@ const login = async (req: Request, res: Response, next: NextFunction) => {
 const googleLogin = async (req: Request, res: Response, next: NextFunction) => {
   try {
     debug(`AuthController: ${chalk.green("authenticate googleLogin")}`);
-    const { idToken } = req.body;
 
-    const userInfo = await authRepository.googleLogin(idToken);
+    const { idToken } = req.body;
     const source: string = uriBuilder(req);
+    const userInfo = await authRepository.googleLogin(idToken);
```

Comment:
this is not a `repository`, this is a `service`

Reply:
Ok, I will fix it that